Bottom: 122c6a6fa8538ddc3b72a2843ce7b6e30ae00681
Top:    281dea9ff424123572c068eacb1debfa0efcd8c1
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-06-29 14:56:53 +0800

Refresh of vhost-introduce-new-ioctls-to

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 23a478e..4a123e8 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -78,7 +78,7 @@ enum {
 };
 
 enum {
-	VHOST_NET_BACKEND_FEATURES = (1ULL << VHOST_BACKEND_F_MSG_V2)
+	VHOST_NET_BACKEND_FEATURES = (1ULL << VHOST_BACKEND_F_IOTLB_MSG_V2)
 };
 
 enum {
diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 25f2cc5..9501d4c 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1154,7 +1154,7 @@ static int vhost_iotlb_miss(struct vhost_virtqueue *vq, u64 iova, int access)
 	struct vhost_dev *dev = vq->dev;
 	struct vhost_msg_node *node;
 	struct vhost_iotlb_msg *msg;
-	bool v2 = vhost_backend_has_feature(vq, VHOST_BACKEND_F_MSG_V2);
+	bool v2 = vhost_backend_has_feature(vq, VHOST_BACKEND_F_IOTLB_MSG_V2);
 
 	node = vhost_new_msg(vq, v2? VHOST_IOTLB_MSG_V2 : VHOST_IOTLB_MSG);
 	if (!node)
diff --git a/include/uapi/linux/vhost.h b/include/uapi/linux/vhost.h
index 613412b..c176e9d 100644
--- a/include/uapi/linux/vhost.h
+++ b/include/uapi/linux/vhost.h
@@ -173,7 +173,7 @@ struct vhost_memory {
 /* Set or get vhost backend capability */
 
 /* Use message type V2 */
-#define VHOST_BACKEND_F_MSG_V2 0x1
+#define VHOST_BACKEND_F_IOTLB_MSG_V2 0x1
 
 #define VHOST_SET_BACKEND_FEATURES _IOW(VHOST_VIRTIO, 0x25, __u64)
 #define VHOST_GET_BACKEND_FEATURES _IOW(VHOST_VIRTIO, 0x26, __u64)
